[2025-07-19T17:40:19.260+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-07-19T17:40:19.281+0000] {taskinstance.py:2603} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: banking_data_quality.generate_data scheduled__2025-07-18T17:00:00+00:00 [queued]>
[2025-07-19T17:40:19.292+0000] {taskinstance.py:2603} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: banking_data_quality.generate_data scheduled__2025-07-18T17:00:00+00:00 [queued]>
[2025-07-19T17:40:19.292+0000] {taskinstance.py:2856} INFO - Starting attempt 4 of 5
[2025-07-19T17:40:19.308+0000] {taskinstance.py:2879} INFO - Executing <Task(PythonOperator): generate_data> on 2025-07-18 17:00:00+00:00
[2025-07-19T17:40:19.317+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=212) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-07-19T17:40:19.319+0000] {standard_task_runner.py:72} INFO - Started process 214 to run task
[2025-07-19T17:40:19.320+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'banking_data_quality', 'generate_data', 'scheduled__2025-07-18T17:00:00+00:00', '--job-id', '8', '--raw', '--subdir', 'DAGS_FOLDER/banking_dq_dag.py', '--cfg-path', '/tmp/tmpvd8y71er']
[2025-07-19T17:40:19.322+0000] {standard_task_runner.py:105} INFO - Job 8: Subtask generate_data
[2025-07-19T17:40:19.376+0000] {task_command.py:467} INFO - Running <TaskInstance: banking_data_quality.generate_data scheduled__2025-07-18T17:00:00+00:00 [running]> on host 894e965ab49e
[2025-07-19T17:40:19.465+0000] {taskinstance.py:3122} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='banking_data_quality' AIRFLOW_CTX_TASK_ID='generate_data' AIRFLOW_CTX_EXECUTION_DATE='2025-07-18T17:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='4' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-07-18T17:00:00+00:00'
[2025-07-19T17:40:19.466+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-07-19T17:40:19.485+0000] {logging_mixin.py:190} INFO - --------connect success---------
[2025-07-19T17:40:59.652+0000] {local_task_job_runner.py:341} WARNING - State of this instance has been externally set to up_for_retry. Terminating instance.
[2025-07-19T17:40:59.653+0000] {local_task_job_runner.py:240} INFO - ::endgroup::
[2025-07-19T17:40:59.654+0000] {process_utils.py:132} INFO - Sending 15 to group 214. PIDs of all processes in the group: [214]
[2025-07-19T17:40:59.654+0000] {process_utils.py:87} INFO - Sending the signal 15 to group 214
[2025-07-19T17:41:59.691+0000] {process_utils.py:150} WARNING - process psutil.Process(pid=214, name='airflow task ru', status='sleeping', started='17:40:19') did not respond to SIGTERM. Trying SIGKILL
[2025-07-19T17:41:59.692+0000] {process_utils.py:87} INFO - Sending the signal 9 to group 214
[2025-07-19T17:41:59.705+0000] {process_utils.py:80} INFO - Process psutil.Process(pid=214, name='airflow task ru', status='terminated', exitcode=<Negsignal.SIGKILL: -9>, started='17:40:19') (214) terminated with exit code -9
[2025-07-19T17:41:59.706+0000] {standard_task_runner.py:190} ERROR - ('Job 8 was killed before it finished (likely due to running out of memory)', 'For more information, see https://***.apache.org/docs/apache-***/stable/troubleshooting.html#LocalTaskJob-killed')
[2025-07-19T18:03:54.213+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-07-19T18:03:54.231+0000] {taskinstance.py:2603} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: banking_data_quality.generate_data scheduled__2025-07-18T17:00:00+00:00 [queued]>
[2025-07-19T18:03:54.242+0000] {taskinstance.py:2603} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: banking_data_quality.generate_data scheduled__2025-07-18T17:00:00+00:00 [queued]>
[2025-07-19T18:03:54.242+0000] {taskinstance.py:2856} INFO - Starting attempt 4 of 5
[2025-07-19T18:03:54.255+0000] {taskinstance.py:2879} INFO - Executing <Task(PythonOperator): generate_data> on 2025-07-18 17:00:00+00:00
[2025-07-19T18:03:54.267+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=140) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-07-19T18:03:54.269+0000] {standard_task_runner.py:72} INFO - Started process 142 to run task
[2025-07-19T18:03:54.269+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'banking_data_quality', 'generate_data', 'scheduled__2025-07-18T17:00:00+00:00', '--job-id', '8', '--raw', '--subdir', 'DAGS_FOLDER/banking_dq_dag.py', '--cfg-path', '/tmp/tmpld26qo37']
[2025-07-19T18:03:54.272+0000] {standard_task_runner.py:105} INFO - Job 8: Subtask generate_data
[2025-07-19T18:03:54.323+0000] {task_command.py:467} INFO - Running <TaskInstance: banking_data_quality.generate_data scheduled__2025-07-18T17:00:00+00:00 [running]> on host 2e145ce6e7b6
[2025-07-19T18:03:54.409+0000] {taskinstance.py:3122} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='banking_data_quality' AIRFLOW_CTX_TASK_ID='generate_data' AIRFLOW_CTX_EXECUTION_DATE='2025-07-18T17:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='4' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-07-18T17:00:00+00:00'
[2025-07-19T18:03:54.410+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-07-19T18:03:54.427+0000] {logging_mixin.py:190} INFO - --------connect success---------
[2025-07-19T18:03:54.492+0000] {logging_mixin.py:190} INFO - customers = [11, 12, 13, 14, 15, 16, 17, 18, 19, 20]
[2025-07-19T18:03:54.502+0000] {logging_mixin.py:190} INFO - accounts = [20, 21, 22, 23, 24, 25, 26, 27, 28, 29, 30, 31, 32, 33, 34, 35, 36, 37, 38, 39]
[2025-07-19T18:03:54.512+0000] {logging_mixin.py:190} INFO - cards = [33, 34, 35, 36, 37, 38, 39, 40, 41, 42, 43, 44, 45, 46, 47, 48, 49, 50, 51, 52, 53, 54, 55, 56]
[2025-07-19T18:03:54.521+0000] {logging_mixin.py:190} INFO - merchants = [11, 12, 13, 14, 15, 16, 17, 18, 19, 20]
[2025-07-19T18:03:54.542+0000] {logging_mixin.py:190} INFO - devices = [18, 19, 20, 21, 22, 23, 24, 25, 26, 27, 28, 29, 30, 31, 32, 33, 34, 35, 36, 37]
[2025-07-19T18:03:54.551+0000] {logging_mixin.py:190} INFO - auth_logs = [21, 22, 23, 24, 25, 26, 27, 28, 29, 30, 31, 32, 33, 34, 35, 36, 37, 38, 39, 40]
[2025-07-19T18:03:54.585+0000] {logging_mixin.py:190} INFO - Sample data generated successfully.
[2025-07-19T18:03:54.586+0000] {logging_mixin.py:190} INFO - Data generation completed successfully.
[2025-07-19T18:03:54.586+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-07-19T18:03:54.596+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-07-19T18:03:54.596+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=banking_data_quality, task_id=generate_data, run_id=scheduled__2025-07-18T17:00:00+00:00, execution_date=20250718T170000, start_date=20250719T180354, end_date=20250719T180354
[2025-07-19T18:03:54.644+0000] {local_task_job_runner.py:261} INFO - Task exited with return code 0
[2025-07-19T18:03:54.674+0000] {taskinstance.py:3891} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-07-19T18:03:54.676+0000] {local_task_job_runner.py:240} INFO - ::endgroup::
